pyrightconfig.json

```diff
@@ -10,5 +10,5 @@
     ],
     "typeCheckingMode": "standard",
     "reportMissingImports": true,
-    "reportMissingTypeStubs": true,
+    "reportMissingTypeStubs": false,
```

Comment:
The web agent PR (#79) added a package which does not have type information or a stub file with types. I've set this to false for now but will circle back in a follow-up PR to add the stub files for the package and then re-enable this.